openeo/extra/job_management/job_splitting.py

```diff
@@ -0,0 +1,136 @@
+import abc
+import math
+from typing import Dict, List, NamedTuple, Optional, Union
+
+import pyproj
+import shapely
+from shapely.geometry import shape
+from shapely.ops import transform
+
+
+class JobSplittingFailure(Exception):
+    pass
+
+
+# TODO: This function is also defined in openeo-python-driver. But maybe we want to avoid a dependency on openeo-python-driver?
```

Comment:
yes, we can not depend on openeo-python-driver from the client (the other way around is already happening, so that would actually be a circular dependency). 
just remove this TODO